src/atomate2/vasp/jobs/lobster.py

```diff
@@ -0,0 +1,301 @@
+    """
+
+    name: str = "static_run"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: StaticSetGenerator(
+            user_kpoints_settings={"grid_density": 6000},
+            user_incar_settings={
+                "IBRION": 2,
+                "ISIF": 2,
+                "ENCUT": 700,
+                "EDIFF": 1e-7,
+                "LAECHG": False,
+                "LREAL": False,
+                "ALGO": "Normal",
+                "NSW": 99,
+                "LCHARG": False,
+                "LWAVE": True,
+                "ISYM": 0,
+            },
+        )
+    )
+    copy_vasp_kwargs: dict = field(
+        default_factory=lambda: {"additional_vasp_files": ["WAVECAR"]}
+    )
+
+
+@job
+def get_basis_infos(
+    structure: Structure,
+    vaspmaker: BaseVaspMaker,
+    address_max_basis: str = None,
+    address_min_basis: str = None,
+):
+    """
+
+    Args:
+        structure: Structure object.
+        vaspmaker: BaseVaspMaker.
+        address_max_basis: string to yaml file including basis set information.
+        address_min_basis: string to yaml file including basis set information.
+
+    Returns:
+        dict including number of bands and basis set information
+    """
+    potcar_symbols = vaspmaker.input_set_generator._get_potcar(
+        structure=structure, potcar_spec=True
+    )
+
+    # get data from Lobsterinput
+    # Lobster
+    if address_max_basis is None and address_min_basis is None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure, potcar_symbols=potcar_symbols
+        )
+    elif address_max_basis is not None and address_min_basis is None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure,
+            potcar_symbols=potcar_symbols,
+            address_basis_file_max=address_max_basis,
+        )
+    elif address_min_basis is not None and address_max_basis is None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure,
+            potcar_symbols=potcar_symbols,
+            address_basis_file_min=address_min_basis,
+        )
+    elif address_min_basis is not None and address_max_basis is not None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure,
+            potcar_symbols=potcar_symbols,
+            address_basis_file_max=address_max_basis,
+            address_basis_file_min=address_min_basis,
+        )
+
+    nband_list = []
+    for dict_for_basis in list_basis_dict:
+        basis = [key + " " + value for key, value in dict_for_basis.items()]
+        lobsterin = Lobsterin(settingsdict={"basisfunctions": basis})
+        nbands = lobsterin._get_nbands(structure=structure)
+        nband_list.append(nbands)
+
+    return {"nbands": max(nband_list), "basis_dict": list_basis_dict}
+
+
+@job
+def update_user_incar_settings_maker(
```

Comment:
Ah! I see you ran into this issue as well. Which is that can't put output references in the class fields for Makers. Kind of annoying and a limitation of jobflow. I might try and fix this in the future.